composer.json

```diff
@@ -34,7 +34,7 @@
         }
     ],
     "require": {
-        "php": "^8.1",
+        "php": "^8.0|^8.1",
```

Comment:
Minimum version for this branch of package is 8.1 and i don't want to decrease it.

Reply:
this is not changed @majidfeiz